src/Broker/ActiveMq/Mode/DurableSubscription.php

```diff
@@ -39,15 +39,17 @@ class DurableSubscription extends ActiveMqMode
      * @param string $topic
      * @param string $selector
      * @param string $ack
+     * @param string $subscriptionId
      * @throws StompException
      */
-    public function __construct(Client $client, $topic, $selector = null, $ack = 'auto')
+    public function __construct(Client $client, $topic, $selector = null, $ack = 'auto', $subscriptionId = null)
     {
         parent::__construct($client);
         if (!$client->getClientId()) {
             throw new StompException('Client must have been configured to use a specific clientId!');
         }
```

Comment:
Make subscription id required for Artemis.
```suggestion
        }
        $server = $client->getProtocol()->getServer();
        if (is_null($subscriptionId) && substr($server, 0, 16) === 'ActiveMQ-Artemis') {
            throw new StompException('Durable subscription requires a specific subscriptionId!');
        }
```